docs/sidebars.js

```diff
@@ -66,6 +66,17 @@ const sidebars = {
             },
           ],
         },
+        {
+          type: 'category',
+          label: 'Admin',
+          items: [
+            {
+              type: 'doc',
+              label: 'Link Strapi Design System',
+              id: 'core/admin/link-strapi-design-system',
```

Comment:
This is a duplicate of L20 :)

Reply:
fixed in e141207f64390267d57c7c6be847604613da50fd